bears/python/VultureBear.py

```diff
@@ -1,25 +1,56 @@
-import re
 from shutil import which
 
 from coalib.bears.GlobalBear import GlobalBear
 from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
 
 
+class Item(str):
+
+    def __new__(cls, name, typ, filename, lineno):
+        item = str.__new__(cls, name)
+        item.typ = typ
+        item.filename = filename
+        item.lineno = lineno
+        return item
+
+
+class Unused(Item):
+
+    def __new__(cls, item, message):
+        item.message = message
+        return item
+
+
+def retrieve_unused(filenames):
+    import vulture
```

Comment:
If the `import vulture` statement is moved at the top of the file, an error occurs:
```
Traceback (most recent call last):
  File "/home/rahul/.virtual_envs/coala_dev/lib/python3.4/site-packages/coalib/bears/Bear.py", line 282, in execute
    return [] if result is None else list(result)
  File "/home/rahul/Desktop/code/coala-bears/bears/python/VultureBear.py", line 68, in run
    unused = retrieve_unused(filenames)
  File "/home/rahul/Desktop/code/coala-bears/bears/python/VultureBear.py", line 34, in retrieve_unused
    vulture = vulture.Vulture()
UnboundLocalError: local variable 'vulture' referenced before assignment
```
:confused: